mesonbuild/modules/_qt.py

```diff
@@ -453,7 +453,7 @@ def _compile_moc_impl(self, state: ModuleState, kwargs: MocCompilerKwArgs) -> T.
         inc = state.get_include_args(include_dirs=kwargs['include_directories'])
         compile_args: T.List[str] = []
         for dep in kwargs['dependencies']:
-            compile_args.extend(a for a in dep.get_all_compile_args() if a.startswith(('-I', '-D')))
+            compile_args.extend(a for a in dep.get_all_compile_args() if a.startswith(('-I', '-F', '-D')))
```

Comment:
This will pass through any `-F.*` options, is that desireable, or is it just `-F`?

If it is I'd probably prefer `if a.startswith(...) or a in {'-F'}`, to be conservative